tests/Makefile.am

```diff
@@ -77,6 +77,7 @@ TESTS +=  \
 	timegenerated-utc-legacy.sh \
 	timereported-utc.sh \
 	timereported-utc-legacy.sh \
+	mmrm1stspace-basic.sh \
 	rawmsg-after-pri.sh \
```

Comment:
the testbench failures indicate that we need to put our new test into a conditional, so that is it only executed when --enable-mmrm1stspace is given.